jobserver/github.py

```diff
@@ -612,7 +607,7 @@ def get_repos_with_branches(self, org):
                 topics = [n["topic"]["name"] for n in repo["repositoryTopics"]["nodes"]]
 
             if "non-research" in topics:
-                continue  # ignore non-research repos
+                continue  # Ignore non-research repos.
```

Comment:
I admire your commitment to the English language, but would suggest that not every comment needs to be a sentence.

If you disagree -- and why wouldn't you? 🙂 -- could I suggest making the typographical changes in their own commit? Doing so would help the reviewer focus on what's important, when their attention is fading.

Reply:
This commit includes only such typographical changes and one substantive change to a docstring. I take your point that the docstring change could have been separated out further into its own commit.